cmd/chainsimulator/main.go

```diff
@@ -152,13 +159,15 @@ func startChainSimulator(ctx *cli.Context) error {
 	}
 
 	numValidatorsMetaShard := ctx.GlobalInt(numValidatorsMeta.Name)
-	if numValidatorsMetaShard < 1 {
```

Comment:
Do **not** delete existing checks. Simply skip them if you are in sovereign mode